libnavigation-ui/src/main/res/values/strings.xml

```diff
@@ -12,6 +12,9 @@
     <!-- Indicates voice instructions are unmuted -->
     <string name="mapbox_unmuted">Unmuted</string>
 
+    <!-- Max Speed Limit -->
+    <string name="max_speed">MAX\n%1$d</string>
```

Comment:
Is `MAX` used in other countries (outside 🇺🇸 )? Not sure if this needs translatable 🤔 

cc @1ec5

Reply:
This needs to be translated. I don't have access to Transifex. Would need help in this case.

Reply:
I was wondering if `MAX` is common in speed limits in other countries outside US. If that's not the case no need to make this translatable. Do you know @1ec5?

Reply:
MUTCD speed limit signs normally have the legend “SPEED LIMIT”, but some jurisdictions change it to “SPEED” (Oregon), “MAXIMUM SPEED” (sometimes in California), or “MAXIMUM” (Canada). In mapbox/mapbox-navigation-ios#2291, I intentionally chose “MAX” because it’s short in English, has a short equivalent in many other languages, and can’t be confused with an indicator of the user’s current speed. (Perhaps a future your-current-speed indicator would put “YOU” there or something.) The MUTCD design is easily recognizable at a glance as long as there’s _some_ text in that position, regardless of the specific legend. Otherwise, Oregon and California wouldn’t get away with their variants.

This string is localizable on iOS. I think it should be localizable on Android with a note to make the translation as short as possible, so it fits in the design.